HIP XX Crowdspot Modifications

```diff
@@ -0,0 +1,190 @@
+Final Appeal- you get one chance for a 2nd human (and different person) to review after the initial submission confirmed gaming. 
+
+
+What other designs have been considered and what is the rationale for not choosing them?
+HIP 66 and fork 551 both sought to 
+
+
+What is the impact of not doing this?
+
+Each week more submissions are added to Crowdspot than are being removed.  The time for a review will grow so long that owners of legit hotspots that get falsely accused will have virtually no recourse.  Being put on the denylist will be equivalent to a perma-ban. 
+
+Each day submissions are added to Crowdspot by already identified gamers asking for a review of the previous review of a farm that was denied 9 months ago, and further bogging down the system.  This revolving door must be stopped and penalties should be enacted for simply bogging down the system.  
+
+
+ The proposals of this HIP do not undermine the need and impact of the denylist as a whole.  There will always be gamers, therefore there will always be a need of policing those bad actors.  On the alternative side there will occasionally be falsely accused persons.  This HIP only seeks to help identify and clear those falsely accused in an efficient manner.  
+
+ Since the beginning of the denylist, in the old Github repository prior to October 2022, we understood that gamers continued gaming for months while the Github addition/removal list grew longer and longer.  When we moved over to the Crowdspot method, the list of additions to the denylist only grew, and still little attention was given to processing those requests.  Only since the implementation of algorithmic detection has the need for manual additions to the denylist become less useful.  Additions to the denylist via manual review are not the primary target of this HIP.  Manual additions are important, however improvements to the algorithm, and additions to the denylist via algorithm detection will continue to make manual additions process less important moving forward.  However, the requests for removal from the denylist make up about 99% of the pending submissions in Crowdspot.  For example, in the month of April, 2023 there were only 18 addition requests compared to around 300 removal requests placed on crowdspot.  In the current state, most of those requests won't be looked at until nearly 2024!
+
+Unresolved Questions
+
+How long is unreasonable to wait for review? 
+How much HNT to stake to hold your place on the list?
+What to do with the 4100 backlogged requests?
+
+It is expected in the discussion process to determine the following variables in this HIP.  These current variables are only intended for the discussion process and are intended to become static once the discussion is finalized. 
+
+var_submission_expiration_time = 60
```

Comment:
I think this should be 30 days

Reply:
There should be allowances for down time and outages in the ELT that the review team uses.  The time decided should be a balance between unacceptable customer service and allowances for keeping the review team as volunteers.